src/librustc_codegen_ssa/base.rs

```diff
@@ -708,6 +709,17 @@ fn assert_and_save_dep_graph(tcx: TyCtxt<'_>) {
          || ::rustc_incremental::save_dep_graph(tcx));
 }
 
+impl From<def_id::CrateNum> for CrateNum {
+    fn from(crate_num: def_id::CrateNum) -> Self {
+        CrateNum(crate_num.as_u32())
+    }
+}
+impl Into<def_id::CrateNum> for CrateNum {
+    fn into(self) -> def_id::CrateNum {
+        def_id::CrateNum::from_u32(self.0)
+    }
+}
```

Comment:
Please move these to `lib.rs` below the definition of `crate::CrateNum`.

Reply:
Done.